configdependabotfile.go

```diff
@@ -0,0 +1,73 @@
+package main
+
```

Comment:
Please rename the file to dependabotconfig.go. When someone tries to find the file, they will be thinking about dependabot, so will be easier to spot that way.